samples/advanced-python/app.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from flask import Flask, redirect, render_template, request
+app = Flask(__name__)
+
+# Supply your subscription key and endpoint by replacing the below values.
+SUBSCRIPTION_KEY = 'YOUR_SUBSCRIPTION_KEY'

```

Comment:
Put subscription key and endpoint in a separate file, and include that file in the python project, but exclude it from git (via .gitignore).